src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -1076,4 +1076,22 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="ColumnHeader_PackageLevel" xml:space="preserve">
     <value>Package Level</value>
   </data>
+  <data name="Label_PackageDetails" xml:space="preserve">
+    <value>Package Details</value>
+  </data>
+  <data name="Error_UnableToLoadReadme" xml:space="preserve">
+    <value>An error occurred while trying to load the README.</value>
+  </data>
+  <data name="Label_Readme_Tab" xml:space="preserve">
+    <value>README</value>
+  </data>
+  <data name="Text_NoReadme" xml:space="preserve">
+    <value>Could not read package README from selected package.
+
+Only the package maintainer can add a README.
+
+If you are not the maintainer, please consider filing an issue or contacting the maintainer to request a README.
+
+For instructions on how to add a README, please visit [aka.ms/nuget/readme](https://aka.ms/nuget/readme)</value>
```

Comment:
We don't want the url to be localized, we could add the text later or localize that part of the string with https://ceapex.visualstudio.com/CEINTL/_wiki/wikis/CEINTL.wiki/8680/Localizability-Commenting-Overview?anchor=commonly-used-functional-commenting-instruction-(locver-rule)

Reply:
added a lock.